Dockerfile

```diff
@@ -18,19 +19,33 @@ RUN yum install -y curl which patch make git \
 # Switch to directory with sources
 WORKDIR /src
 
-RUN groupadd -g 1003 updater && \
-    useradd -r -u 1003 -g updater updater && \
-    mkhomedir_helper updater && \
-    chown updater: /home && \
-    chown updater: /src
-USER 1003
+# Install + verify RVM with gpg (https://rvm.io/rvm/security)
+RUN gpg2 --quiet --no-tty --logger-fd 1 --keyserver hkp://keys.gnupg.net \
+         --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 \
+                     7D2BAF1CF37B13E2069D6956105BD0E739499BDB \
+    && echo 409B6B1796C275462A1703113804BB82D39DC0E3:6: | \
+       gpg2 --quiet --no-tty --logger-fd 1 --import-ownertrust \
+    && curl -sSO https://raw.githubusercontent.com/rvm/rvm/${RVM_VERSION}/binscripts/rvm-installer \
+    && curl -sSO https://raw.githubusercontent.com/rvm/rvm/${RVM_VERSION}/binscripts/rvm-installer.asc \
+    && gpg2 --quiet --no-tty --logger-fd 1 --verify rvm-installer.asc \
+    && bash rvm-installer ${RVM_VERSION} \
+    && rm rvm-installer rvm-installer.asc \
+    && echo "bundler" >> /usr/local/rvm/gemsets/global.gems \
+    && echo "rvm_silence_path_mismatch_check_flag=1" >> /etc/rvmrc \
+    && echo "install: --no-document" > /etc/gemrc
 
-RUN gpg2 --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
-RUN curl -sSL https://get.rvm.io | bash -s stable
+# Switch to a bash login shell to allow simple 'rvm' in RUN commands
+SHELL ["/bin/bash", "-l", "-c"]
 
-# login shell is required by rvm
-RUN /bin/bash -l -c ". /home/updater/.rvm/scripts/rvm && rvm install jruby-9.2.5.0 && gem update --system \
-    && gem install bundler rake"
+RUN groupadd -g 48 apache \
+    && useradd -u 48 -m --no-log-init -r -g apache -G rvm apache
+
+RUN chown apache: /src
+
+RUN rvm install jruby-9.2.5.0 && gem update --system \
```

Comment:
it might be good to pull the ruby version out to a variable

Reply:
done